### DISCUSSION TITLE
左下角 123 旁边的地球怎么弄没啊

### DISCUSSION THREAD

**Discussion by dnxxymcj** [imfuxiao/Hamster#188] [Q&A]
![image](https://user-images.githubusercontent.com/7660888/235349111-dfd09191-d43c-4696-b98a-f67df8e4c9dd.png)

**Comment by imfuxiao** [comment]
您好，目前在非全面屏下是强制显示语言切换按键的，因为系统键盘在某些情况下还是需要的。

不过后续可以增加一个配置选项，让不喜欢此切换按钮的伙伴可以关闭。

**Reply by dnxxymcj** [reply]
好的，谢谢

**Comment by DreamAfar** [comment]
可以增加一个选项关闭显示地球按键，上下滑动加个切换系统输入功能的选项，然后通过上下滑动来切换（类似i键的切换中英文）。

**Reply by yuantongziran** [reply]
> 可以增加一个选项关闭显示地球按键，上下滑动加个切换系统输入功能的选项，然后通过上下滑动来切换（类似i键的切换中英文）。

不给键盘全部访问权限，地球就不显示
